Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM mhart/alpine-node:6
+MAINTAINER code.vire@gmail.com
+
+RUN npm install -g yarn
+
+COPY package.json /tmp
+RUN cd /tmp &&\
+    yarn &&\
+    mkdir /app &&\
+    mv node_modules /app/node_modules &&\
+    rm package.json
+
+WORKDIR /app
+COPY . /app
```

Comment:
I wasn't sure about putting yarn.lock right there until 1.0 of yarn, but ths comment conviced me!